assets/yq-for-mixs_subset_modified.txt

```diff
@@ -147,6 +147,7 @@
 '.slots.gravity.inlined_as_list |= true'
 '.slots.growth_hormone_regm.inlined_as_list |= true'
 '.slots.heavy_metals.inlined_as_list |= true'
+'.slots.heavy_metals_meth.inlined_as_list |= true'
```

Comment:
meth.*range values are a mixture of string and TextValue

Reply:
in assets/yq-for-mixs_subset_modified.txt

- '.slots.biocide_admin_method.range |= "TextValue"'
- '.slots.chem_treat_method.range |= "string"'
- '.slots.heat_sys_deliv_meth.range |= "string"'
- '.slots.micro_biomass_meth.range |= "string"'
- '.slots.prev_land_use_meth.range |= "string"'
- '.slots.samp_collec_method.range |= "string"'
- '.slots.soil_texture_meth.range |= "string"'
- '.slots.tot_nitro_cont_meth.range |= "string"'
- '.slots.water_cont_soil_meth.range |= "string"'
-